### ISSUE TITLE
Add info that says "setting alternate volume to zero will mute audio during unthrottle"

### ISSUE THREAD

**Issue by radityaargap** [hrydgard/ppsspp#14736]
### What should happen

Just a little feedback. I was messing with ppsspp settings and I found out that setting "alternate speed volume" to zero mutes audio during unthrottle. I never knew what this setting does, so I always leave it as default, which is global.

It's a nice feature to have because the screechy noise can be irritating. Retroarch and Snes9x call this feature "mute during fast forward". It would be better to use the same familiar term or add an info that explains what the setting does.

### Who would this benefit

Any

### Games this would be useful in

Any

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by hrydgard** [comment]
I'm thinking about changing the word "unthrottle" to "fast forward" all over the interface, because it is more intuitive, even though unthrottle can be slightly more accurate (disabling the speed throttle just letting it go as fast as it can). In reality unthrottle in many situations is limited to 8x anyway (frameskipping unthrottle).

**Comment by unknownbrackets** [comment]
I'll note that this setting changes the volume whenever the speed isn't normal.  For example, if you set alternate speed to 50%, this also affects the volume when switching to 50% speed.  I personally like it that way, and I don't really care if that doesn't conform to the more limited options of some other emulator.

-[Unknown]

**Comment by hrydgard** [comment]
Yeah I agree with @unknownbrackets , the semantics of the option are fine as-is, just some wording is confusing. Shouldn't have tagged this to be done, as what I'm going to change (the wording) isn't what this issue is asking fore.
